app/views/shared/summary_cards/_group.html.slim

```diff
@@ -21,13 +21,18 @@
           = humanize_date(resource.published_at)
       .summary-card__metadata
         p
+          span.glyphicon.glyphicon-comment.glyphicon--right
+          span.disqus-comment-count data-disqus-identifier="#{url_for(resource)}" 
+            | 0 Comments
```

Comment:
> It also includes the comments count in the summary cards for these types of records.

Is this the line that's supposed to do it? I've added a comment on a resource locally but I'm not seeing the count increase...